_posts/ianmoore/2016-03-22-natalia-and-ian-file.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+author: touchwick
+title: "Ian and Natalia's File Exercises"
+---
+This is exercise 1, which Ian completed:
+
+<iframe src="https://trinket.io/embed/python3/a0ab418311" width="100%" height="356" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This is exercise 2, which Ian was unable to get to work successfully:
+
+<iframe src="https://trinket.io/embed/python/b5a789e422" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
```

Comment:
@touchwick first of all this should be a python3 trinket and it's a python trinket.  thw `with` statement only works in Python 3.

secondly, the .split() method will work but since you know the length of things you can probably use .startswith() and a slice to get the data you need.